internal/handlers/sqlite/msg_find.go

```diff
@@ -80,6 +80,32 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 		qp.Filter = params.Filter
 	}
 
+	// Skip sorting if there are more than one sort parameters
```

Comment:
Why is there no change related related to limit? 
Shouldn't it use newly added `GetLimitParam()`, also some test should fail right?

Reply:
`GetLimitParams` is used from `common/GetExplainParams()`.

Reply:
I think naming is confusing, exported `GetLimitParams` parameter indicates it should be use for limit parameters. We should change godoc and function name.

Reply:
My question is why is there no limit pushdown related change in this file? `msg_find.go` should set query limit parameter right? :thinking: Could we add a test for it if it's easy to miss?